tools/spec_to_manifest.py

```diff
@@ -0,0 +1,189 @@
+import json
+import re
+import sys
+import yaml
+import logging
+
+
+def main():
+    filename = ""
+    path = ""
+    correct_params = False
+    try:
+        filename = sys.argv[1]
+        path = sys.argv[2]
+        correct_params = True
+    except IndexError:
+        logging.warning('ERROR: Insufficient parameters')
+
+    if correct_params:
+        if verify_yaml(filename):
+            spec = open_and_read_spec(filename)
+            print("Read spec file at " + filename)
+            json_obj = parse_fields(spec)
+            print("Converted fields")
+            write_to_json(json_obj, path)
+            print("Wrote manifest at " + path)
+        else:
+            logging.warning("ERROR: Wrong file type specified")
+
+
+def parse_fields(yaml_obj):
+    # Manifest Version -- mandatory
+    manifest_obj = {"manifest_version": 1}
+
+    # Extension -- mandatory
+    # In this case, only workflows
+    extension_type = yaml_obj.get("extension")
+    extension_obj = {"type": extension_type, extension_type: yaml_obj.get("name") + ".icon"}
```

Comment:
The .icon filename is only needed if it's a workflow spec, so test that the spec is a workflow spec first